tests/fail-dep/libc/fs/unix_open_missing_required_mode.rs

```diff
@@ -8,5 +8,5 @@ fn main() {
 fn test_file_open_missing_needed_mode() {
     let name = b"missing_arg.txt\0";
     let name_ptr = name.as_ptr().cast::<libc::c_char>();
-    let _fd = unsafe { libc::open(name_ptr, libc::O_CREAT) }; //~ ERROR: Undefined Behavior: incorrect number of arguments for `open` with `O_CREAT`: got 2, expected at least 3
+    let _fd = unsafe { libc::open(name_ptr, libc::O_CREAT) }; //~ ERROR: Undefined Behavior: incorrect number of arguments for `open(pathname, O_CRETE, ...)`: got 2, expected at least 3
```

Comment:
`O_CRETE` looks wrong?

Reply:
A typo multiplied by copy-paste.

Squash after addressing all the three comments?

Reply:
> Squash after addressing all the three comments?

Yes